shark_turbine/kernel/wave/docs/gemm_example.md

```diff
@@ -0,0 +1,153 @@
+# Matrix Multiplication
+
+Let's assume we want to compute the following matrix multiplication.
+$$ C = A \times B^{T} $$
```

Comment:
Formatting is broken here: https://github.com/harsh-nod/iree-turbine/blob/dox/shark_turbine/kernel/wave/docs/gemm_example.md

Reply:
thanks, will fix.